File: src/main/java/turniplabs/industry/block/BlockOreTin.java
package turniplabs.industry.block;

import net.minecraft.src.Block;
import net.minecraft.src.Material;
import turniplabs.industry.ModIndustry2;

import java.util.Random;

public class BlockOreTin extends Block {

    public BlockOreTin(int id, Material material) {
        super(id, material);
    }

    @Override
    public int idDropped(int i, Random random) {
        return ModIndustry2.RAW_TIN.itemID;
    }
}

File: src/main/kotlin/turniplabs/industry/block/BlockOreTin.kt
package turniplabs.industry.block

import net.minecraft.src.Block
import net.minecraft.src.Material
import turniplabs.industry.item.IndustryItems
import java.util.*

class BlockOreTin(i: Int, material: Material?) : Block(i, material) {

    override fun idDropped(i: Int, random: Random?): Int {
        return IndustryItems.RAW_TIN.itemID
    }
}